modules/heat_conduction/test/tests/multiple_contact_pairs/3blk.jou

```diff
@@ -0,0 +1,41 @@
+reset
+
```

Comment:
I do not see any `jou` files been added to MOOSE. Is it possible to use moose build in mesh generator? If not, I think we just add `exodus` file alone.

Reply:
Removed the journal file. 
Sometimes we have added journal files to avoid adding overhead when maintaining tests or creating new ones.

Reply:
I prefer to check in the journal files. They're generally pretty small, and they save us a lot of trouble if we ever have to go back and modify the mesh. I've had to re-create those many times when they weren't checked in.

Reply:
Actually I was wrong. There are some `jou` files in MOOSE. So, I am fine with it to add it. @recuero Sorry for reverting your work.

Reply:
No problem, Wen, added it back.

Reply:
Yea I think `*.jou` files should *always* be added when the input mesh is produced via cubit